packages/@aws-cdk/aws-apigateway/lib/method.ts

```diff
@@ -173,6 +179,8 @@ export class Method extends cdk.Construct {
       requestTemplates: options.requestTemplates,
       passthroughBehavior: options.passthroughBehavior,
       integrationResponses: options.integrationResponses,
+      connectionType: options.connectionType,
+      connectionId: options.vpcLink ? options.vpcLink.vpcLinkId : undefined,
```

Comment:
The docs for ConnectionId state:

> The ID of the VpcLink used for the integration when connectionType=VPC_LINK and undefined, otherwise. 

I took that to mean that it would just be ignored. Probably best to throw an error though to alert the user, since that's probably not want they wanted to happen.

Reply:
That's what I was getting at. Thanks.